ros_gz_bridge/include/ros_gz_bridge/ros_gz_bridge.hpp

```diff
@@ -68,6 +68,9 @@ class RosGzBridge : public rclcpp::Node
 
   /// \brief Timer to control periodic callback
   rclcpp::TimerBase::SharedPtr heartbeat_timer_;
+
+  /// \brief Whether the config_file parameter was parsed or not.
+  bool config_file_parsed_;
```

Comment:
nit: This can be initialized here and not in the constructor
```suggestion
  bool config_file_parsed_{false};
```

Reply:
Updated in [13f4008](https://github.com/gazebosim/ros_gz/pull/511/commits/13f40087d99b92c5bc4c8b157a951aad1efc0d9f)